### DISCUSSION TITLE
Using REST API to Create an Event: How do I set the payment method?

### DISCUSSION THREAD

**Discussion by bonesclarke** [pretix/pretix#4677] [Q&A]
I am able to create an event, set the tax rule, and then set the items in accordance to the tax rule using the respective API endpoints, but I am not sure how to set the Stripe payment method for an event. I load in the Stripe plugin in the post create an event API call, but is there a way to set Stripe as the payment method using the Rest APIs?

**Comment by raphaelm** [comment]
It's not possible unfortunately since we can't really implement the complex flow of all the payment providers as a generic API.

The best workaround is creating the first event from hand and then use the API co clone it.
